conf/default-vars.yaml

```diff
@@ -0,0 +1,35 @@
+vars:
+  #families
+  font-primary: 'sans-serif'
+  font-secondary: 'serif'
+  font-heading: 'serif'
```

Comment:
Should we also just pick the generically available fonts like Arial and create a prettier font stack that can be overridden?  Or do we get super fancy and add the system font declarations?